fig-files/validation-spec/2024-validations.csv

```diff
@@ -1,80 +1,82 @@
+W0842,business_ownership_status.duplicates_in_field,business_ownership_status,duplicates_in_field,Warning,Field,business_ownership_status,* 'Business ownership status' should **not** contain duplicated values.,NA
+W0843,business_ownership_status.multi_value_field_restriction,business_ownership_status,multi_value_field_restriction,Warning,Field,business_ownership_status,"* When 'business ownership status' contains 966 (the applicant responded that they did not wish to provide this information) or 988 (not provided by applicant), 'business ownership status' should **not** contain more than one value.",NA
+E0860,num_principal_owners_flag.invalid_enum_value,num_principal_owners_flag,invalid_enum_value,Error,Field,num_principal_owners_flag,* 'Number of principal owners: NP flag' must equal 900 or 988.,NA
+E0880,num_principal_owners.invalid_enum_value,num_principal_owners,invalid_enum_value,Error,Field,num_principal_owners,"* When present, 'number of principal owners' must equal 0, 1, 2, 3, or 4.",NA
+W0901,po_1_ethnicity.duplicates_in_field,po_1_ethnicity,duplicates_in_field,Warning,Field,po_1_ethnicity,* 'Ethnicity of principal owner 1' should **not** contain duplicated values.,NA
+E0900,po_1_ethnicity.invalid_enum_value,po_1_ethnicity,invalid_enum_value,Error,Field,po_1_ethnicity,"* When present, each value in 'ethnicity of principal owner 1' (separated by semicolons) must equal 1, 11, 12, 13, 14, 2, 966, 977, or 988.",NA
+W0902,po_1_ethnicity.multi_value_field_restriction,po_1_ethnicity,multi_value_field_restriction,Warning,Field,po_1_ethnicity,"* When 'ethnicity of principal owner 1' contains 966 (the applicant responded that they did not wish to provide this information) or 988 (not provided by applicant), 'ethnicity of principal owner: 1' should **not** contain more than one value.",NA
 E2040,po_1_ethnicity_ff.conditional_field_conflict,po_1_ethnicity_ff,conditional_field_conflict,Error,Row,"po_1_ethnicity;
-po_1_ethnicity_ff","When 'ethnicity of principal owner 1' does not contain 977 (the applicant responded in the free-form text field), 'ethnicity of principal owner 1: free-form text field for other Hispanic or Latino' must be blank. When 'ethnicity of principal owner 1' contains 977, 'ethnicity of principal owner 1: free-form text field for other Hispanic or Latino' must not be blank.","IF po_1_ethnicity does not contain 977 THEN
+po_1_ethnicity_ff","* When 'ethnicity of principal owner 1' does **not** contain 977 (the applicant responded in the                         free-form text field), 'ethnicity of principal owner 1: free-form text field for other Hispanic                         or Latino' must be blank.
+* When 'ethnicity of principal owner 1' contains 977, 'ethnicity of principal owner 1: free-form                         text field for other Hispanic or Latino' must **not** be blank.
+","IF po_1_ethnicity does not contain 977 THEN
```

Comment:
Why the extra spaces? Is the `dedent()` function not working as-expected? Or is it working, but the way the Python/CSV compare tests work, it needs the spaces?

I don't think we want these extra spaces. The description field now isn't really Markdown we'd want to hand over to anyone. So...I guess if `dedent()` isn't working, we should roll it back to what you had. If it's the tests, can we refactor those so it can do a compare against the `dedent()`ifield Python to the previous no-extra-spaces version of the CSV?

Apologies if going down the `dedent()` route is being a pain.

Reply:
I responded to this in MM, but the jist is, in python, breaking very long sentences up with \ is causing this, unless I remove indentation but that will make the python code ugly.  The \'s were added so you didn't have one line scrolling off the IDE page. However, it's causing the csv to include those.  

In the markdown, it actually renders fine.  It just now looks uglier in here. https://github.com/cfpb/sbl-filing-api/wiki/Example-Markdown-Descriptions-from-Validation-Results has the new markdown after the dedent.  If you look at the markdown itself, it has those tab spaces.  However, rendered in a page, it's fine.